src/snmp_core.cc

```diff
@@ -389,7 +389,7 @@ snmpDecodePacket(SnmpRequest * rq)
         return;
     }
 
-    debugs(49, 5, HERE << "Called.");
+    debugs(49, 5, "Called.");
```

Comment:
BTW, the biggest problem with the "hanging `:`" is not that it looks a little odd but is that the reader cannot tell whether something was meant to be printed there but was not due to bugs or being empty. Printing `;` would address that problem. We could also print `.` but that feels too "final" in this context to me. And `;` also follows programming languages paradigm that fits well here.